Warp.WebApp/Services/Entries/IEntryService.cs

```diff
@@ -7,6 +7,6 @@ namespace Warp.WebApp.Services.Entries;
 public interface IEntryService
 {
     public Task<Result<Guid, ProblemDetails>> Add(Guid userId, string content, TimeSpan expiresIn, List<Guid> imageIds, CancellationToken cancellationToken);
-    public Task<Result<EntryInfo, ProblemDetails>> Get(Guid userId, Guid entryId, CancellationToken cancellationToken, bool isReceivedForCustomer = false);
+    public Task<Result<EntryInfo, ProblemDetails>> Get(Guid userId, Guid entryId, CancellationToken cancellationToken, bool isReceivedForCustomer = false, Guid? customerGuid = null);
```

Comment:
Conventionally, `CancellationToken` should be the last parameter of a method. Probably, we have to add a default value to it